CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - JSON API: `listfunds` now lists a blockheight for confirmed transactions
+- JSON API: `fundchannel_start` now includes field `scriptpubkey`
```

Comment:
What about "The return of `fundchannel_start` now includes field `scriptpubkey` " ?

Reply:
good point. updated to make a bit clearer.